tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/generators/cpp/CppProtocolTestGenerator.java

```diff
@@ -157,15 +159,48 @@ private List<String> computeTestSourceIncludes(ProtocolTestSuite testSuite) {
                         }
 
                     }
-                    // include the request shape itself
+                    return includes;
+                })
+                .flatMap(entrySet -> entrySet.stream())
+                .collect(Collectors.toSet());
+        // include the request shapes
+        Set<String> requestShapes = testSuite.getCases().stream()
+                .map(entry -> {
```

Comment:
looks like we only ever add one item to the set ever, simplified this could just be
```java
testSuite.getCases().stream()
  .map(entry -> { return String.format("<aws/%s/model/%s.h>", 
      serviceModel.getMetadata().getProjectName(),
      entry.getGiven().getName() + "Request")})
  .collect(Collectors.toSet());
````
to avoid creating a extra set per test case

Reply:
copy-paste mistake, thank you!